x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/analytics_list/analytics_list.tsx

```diff
@@ -99,16 +102,22 @@ export const DataFrameAnalyticsList: FC<Props> = ({
 
   // Query text/job_id based on url but only after getAnalytics is done first
   // selectedJobIdFromUrlInitialized makes sure the query is only run once since analytics is being refreshed constantly
-  const [selectedJobIdFromUrlInitialized, setSelectedJobIdFromUrlInitialized] = useState(false);
+  const [selectedIdFromUrlInitialized, setSelectedIdFromUrlInitialized] = useState(false);
   useEffect(() => {
-    if (selectedJobIdFromUrlInitialized === false && analytics.length > 0) {
-      const selectedJobIdFromUrl = getSelectedJobIdFromUrl(window.location.href);
-      if (selectedJobIdFromUrl !== undefined) {
-        setSelectedJobIdFromUrlInitialized(true);
-        setSearchQueryText(selectedJobIdFromUrl);
+    if (selectedIdFromUrlInitialized === false && analytics.length > 0) {
+      const { jobId, groupIds } = getSelectedIdFromUrl(window.location.href);
```

Comment:
We should not access `window.location.href` directly. Would be great to receive it from the router state instead

Reply:
👍  To be done in a follow up.